### DISCUSSION TITLE
`path` argument of "List repository commits" is overriden

### DISCUSSION THREAD

**Discussion by elongl** [python-gitlab/python-gitlab#2845] [Q&A] [answered]
## Description of the problem, including code/CLI snippet

I'm unable to use the `path` argument of [List repository commits](https://docs.gitlab.com/ee/api/commits.html#list-repository-commits) because it's being [overridden by the library](https://github.com/python-gitlab/python-gitlab/blob/main/gitlab/mixins.py#L243).

Is there a workaround for this? Currently planning on using the `http_list`.

**Comment by JohnVillalovos** [comment, marked as answer]
From: https://python-gitlab.readthedocs.io/en/stable/api-usage.html

You can use the `query_parameters` argument to send arguments that would conflict with python or python-gitlab when using them as kwargs:

```
gl.user_activities.list(from='2019-01-01', iterator=True)  ## invalid

gl.user_activities.list(query_parameters={'from': '2019-01-01'}, iterator=True)  # OK
```
